### DISCUSSION TITLE
tools pod doesn't have cephfs-top utility

### DISCUSSION THREAD

**Discussion by gaord** [rook/rook#12107] [Q&A]
cephfs-top is useful for perf debugging. Hope it is packed into the tool pod.

**Comment by galexrt** [comment]
Rook uses the `quay.io/ceph/ceph` images for the daemons and the toolbox pod, so we would need to get the Ceph container team to add `cephfs-top` to the image.

@travisn @gauravsitlani do you know who to ping for considering adding a package to the Ceph container images?

**Reply by Madhu-1** [reply]
@galexrt there is already an opened issue for this one https://github.com/ceph/ceph-container/issues/2079, we just need to follow up or ping the maintainer in that repo
